addons/binding/org.openhab.binding.modbus/ESH-INF/thing/bridge-serial.xml

```diff
@@ -79,6 +79,11 @@
 					<option value="bin">BIN</option>
 				</options>
 			</parameter>
+			<parameter name="enableDiscovery" type="boolean">
```

Comment:
Please mention this in thing configuration section in docs

Reply:
done